src/extension.ts

```diff
@@ -39,6 +39,15 @@ export async function activate(context: vscode.ExtensionContext) {
       );
     }
   );
+  vscode.commands.registerCommand(
+    "templateTree.copyContext",
+    (node: TemplateItem) => {
+      vscode.env.clipboard.writeText(node.copy());
+      vscode.window.showInformationMessage(
+        `Copied AQL of ${node.displayLabel}.`
```

Comment:
Display more appropriate message: "Copied Context Snippets of" would be a start.